doc/guides/spatial/guide_spatially_structured_networks.rst

```diff
@@ -1416,61 +1415,123 @@ center neuron are shown, as well as mask and connection probability.
     :start-after: #{ vislayer #}
     :end-before: #{ end #}
 
-.. TODO: adding masks need to be updated for 3.0?
+.. _ch:custom_masks:
+
+Creating custom masks
+---------------------
+
+In some cases, the built-in masks may not meet your needs, and you want to
+create a custom mask. There are two ways to do this: the first is to use Parameters
+to introduce a cut-off to the connection probability. The second is to implement a custom
+mask in C++ as a module.
+
+Using Parameters is the most accessible option; the entire implementation is done on
```

Comment:
Could we write ``parameters`` in lowercase throughout the guide? That's the letter case we have also used in the "NEST 3.0: What's new?" guide.

Reply:
Done.